pkg/cmd/pipeline/translate.go

```diff
@@ -228,27 +228,23 @@ func updateCmap(cmap *apiv1.ConfigMap, data *CfgData) error {
 }
 
 // AddDevAnnotations add deploy labels to the deployments/sfs
-func AddDevAnnotations(ctx context.Context, manifest *model.Manifest, services []string, c kubernetes.Interface) error {
+func AddDevAnnotations(ctx context.Context, manifest *model.Manifest, c kubernetes.Interface) {
 	repo := os.Getenv(model.GithubRepositoryEnvVar)
-	servicesSet := map[string]bool{}
```

Comment:
@AgustinRamiroDiaz if `services` is empty, which means deploying everything, we were never adding dev annotations, and we should, right?

Reply:
I think it should be already done by this stage. Am I wrong?

Reply:
Once we run `setDeployOptionsValuesFromManifest`, that field should not be empty: https://github.com/okteto/okteto/blob/334b4fa2d108a732849d161014f6660fa29e1857/cmd/deploy/deploy.go#L462-L468 . Maybe the problem is that it's only ran when we have a compose?

Reply:
yes, I was using an okteto manifest and it was empty. But I think it's fine to try all the dev containers, they could be even created by a manual command

Reply:
My point is why to filter the dev containers mapping them with services to deploy. We can always try to annotate all of the dev containers. That filter is broken in a few cases:
- non compose deployments
- autocreate
- dev containers using `selector`

Reply:
Oh yeah, I posted the last comment and hadn't read yours. I agree, could we add some comments/tests for this function?

Also, is there a case where we'd want to fail given that we cannot annotate the dev containers?

Reply:
I removed all errors and changed them to debug info messages because these annotations are optional. We should never return an error if there is an issue annotation a dev container.

Reply:
@AgustinRamiroDiaz added a unit test